react-frontend/src/components/Home/bcLivesteam.js

```diff
@@ -0,0 +1,133 @@
+
+const BCLivestream = () => {
+  const strapiMediaUrl = useContext(AppConfigContext)['state'][
+    'strapiMediaUrl'
+  ];
+  return (
+    <Query query={LIVESTREAM_QUERY}>
+      {({ data: { digitalLivestreamContact, digitalBcLivestreams } }) => {
+        const imageSource =
+          strapiMediaUrl + digitalBcLivestreams[0]?.EventImage?.url;
+        // if no next event is provided in strapi, this section will not render
+        const now = new Date();
+        const nextEventTime = new Date(digitalBcLivestreams[0]?.BroadcastTime);
+
+        {
+          /* Only render if link provided */
+        }
+        const pastBroadcastLink = digitalLivestreamContact?.PastEpisodeURL && (
+          <div>
+            <a href={digitalLivestreamContact?.PastEpisodeURL}>
+              <p style={{ marginBottom: '4px' }}>Watch past broadcasts</p>
+              <img
+                src={youTubeLogo}
+                style={{ height: '20px', width: '80px' }}
+              />
+            </a>
+          </div>
+        );
+
+        const subscriptionLink = digitalLivestreamContact?.SubscriptionURL && (
+          <>
+            <Heading style={{ lineHeight: '1.2' }}>
+              #DigitalBC Livestream
+            </Heading>
+            <p>
+              Watch live broadcasts of monthly updates, webinars, sprint
+              reviews, and more!
+            </p>
+            <LinkExternalButton
+              href={digitalLivestreamContact?.SubscriptionURL}
+              style={{
+                maxWidth: '160px',
+                marginLeft: 'auto',
+                marginRight: 'auto',
+              }}
+            >
+              Subscribe
+            </LinkExternalButton>
+          </>
+        );
+
+        if (nextEventTime && nextEventTime > now) {
+          return (
+            <div style={{ backgroundColor: '#fff' }}>
+              <PageContainer
+                style={{ backgroundColor: '#fff', paddingTop: '40px' }}
+              >
+                <Row between="xs">
+                  <Col
+                    sm={12}
+                    md={4}
+                    style={{ textAlign: 'center', paddingBottom: '20px' }}
+                  >
+                    {subscriptionLink}
+                  </Col>
+                  <Col sm={12} md={4} style={{ paddingBottom: '20px' }}>
+                    <img
+                      src={imageSource}
+                      style={{
+                        width: '100%',
+                        paddingLeft: '10px',
+                        paddingRight: '10px',
+                      }}
+                    />
+                  </Col>
+                  <Col sm={12} md={4} style={{ textAlign: 'center' }}>
+                    <p style={{ marginBottom: '4px' }}>
+                      <b>Next broadcast</b>
+                    </p>
+                    <p
+                      style={{
+                        marginBottom: '8px',
```

Comment:
Could you please replace this `marginBottom` with `margin: '0 auto 8px`

(This accompanies the comment just below ... this will work with `width: 'fit-content'` so that the grey background doesn't spread across the entire width of its container.